From 55b670a5b296825979e81b3e7ee5db5597faab65 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Fri, 15 Nov 2024 10:58:05 +0200
Subject: [PATCH] [FSW-11350] fs1: remove legacy flash offset

From gaudi3 we have hardcoded flash offset to
locate manifest from spi flash. In fs1 we
started using start using start of spi flash
offset for manifest 1, no more hardcoded legacy
offset to locate menifest. Hence removed it.

Change-Id: If91083267b94a3208a56fef973b63e1398102cfc
---

diff --git a/Makefile b/Makefile
index e7d77eb..63f8ff5 100644
--- a/Makefile
+++ b/Makefile
@@ -41,7 +41,6 @@
 BOOT_ROM_CFLAGS += -I$(SPECS_DIR)
 BOOT_ROM_CFLAGS += -I$(SPECS_EMBEDDED_DIR)/fs/common
 BOOT_ROM_CFLAGS += -I$(SPECS_EMBEDDED_DIR)/fs/$(SPECS_EMB_DEV)
-BOOT_ROM_CFLAGS += -DFLASH_START_OFFSET=0x60000
 
 BOOT_ROM_CFLAGS_IO = $(BOOT_ROM_CFLAGS)
 BOOT_ROM_CFLAGS_IO += -DDIE_TYPE=$(DIE_TYPE_IO)
